File: CSC2150/Classes and Objects/Car.java
public class Car
{
    private String make;
    private String model;
    int speed;

    public String getMake() {
        return make;
    }

    public String getModel() {
        return model;
    }

    public int getSpeed() {
        return speed;
    }

    public void setMake(String make) {
        this.make = make;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public void accelerate() {
        speed++;
    }

    public void decelerate() {
        speed--;
    }

    public void accelerate(int increase) {
        speed += increase;
    }

    public static void main(String[] args)
    {
        Car car1 = new Car();
        car1.setMake("Toyota");
        car1.setModel("Corolla");
        System.out.println("The car make is " + car1.getMake());
        System.out.println("The car model is " + car1.getModel());
        for(int i=0; i<10; i++)
            car1.accelerate();
        System.out.println("The car speed is " + car1.getSpeed());
        car1.decelerate();
        car1.decelerate();
        System.out.println("The car speed is " + car1.getSpeed());
        car1.accelerate(20);
        System.out.println("The car speed is " + car1.getSpeed());
    }
}

File: CSC 1700 GitHub/Classes and Objects/car.py
class Car:
    def __init__(self, make, model, year):
        """Initialize the car's attributes"""
        self.__make = make
        self.__model = model
        self.__year = year

    # Setter methods
    def set_make(self, make):
        """Set the car's make"""
        self.__make = make

    def set_model(self, model):
        """Set the car's model"""
        self.__model = model

    def set_year(self, year):
        """Set the car's year"""
        self.__year = year

    # Getter methods
    def get_make(self):
        """Return the car's make"""
        return self.__make

    def get_model(self):
        """Return the car's model"""
        return self.__model

    def get_year(self):
        """Return the car's year"""
        return self.__year


# Program to create a Car object and display its data
def main():
    # Creating a Car object
    my_car = Car("Toyota", "Corolla", 2020)

    # Displaying the car's details using accessor methods
    print("\nCar's Details:")
    print(f"Make: {my_car.get_make()}")
    print(f"Model: {my_car.get_model()}")
    print(f"Year: {my_car.get_year()}")


# Run the main program
if __name__ == "__main__":
    main()
